iter_test.go

```diff
@@ -9,23 +9,23 @@ import (
 )
 
 func TestIterEmpty(t *testing.T) {
-	tq := testQuery{{nil, ListMeta{}, nil}}
+	tq := testQuery{{nil, &ListMeta{}, nil}}
```

Comment:
These all changes so that I can have `testQuery` take a `ListContainer` instead of `ListMeta`. `ListContainer` is an interface, so it requires a pointer.